gserver/test/data/features/after.format.feature

```diff
@@ -7,10 +7,10 @@ Feature: Formatting feature
 	Background: Background name
 
 		Given the following users exist:
-			| name   | email              | twitter         |
-			| Aslak  | aslak@cucumber.io  | @aslak_hellesoy |
-			| Julien | julien@cucumber.io | @jbpros         |
-			| Matt   | matt@cucumber.io   | @mattwynne      |
+			| name   | email              | twitter         | CJK Message 訊息    |
+			| Aslak  | aslak@cucumber.io  | @aslak_hellesoy | Yoo Hello world     |
+			| Julien | julien@cucumber.io | @jbpros         | Yoo 你好 世界       |
+			| Matt   | matt@cucumber.io   | @mattwynne      | Yoo こんにちは 世界 |
```

Comment:
Looks like, that formatting still be wrong - total string length still be different in the last column:
<img width="200" alt="Screen Shot 2019-07-06 at 1 36 25 PM" src="https://user-images.githubusercontent.com/5772406/60755057-2352a980-9ff3-11e9-9128-ddcd31f871c0.png">